data/tutorials/gs_00_up_and_running.md

```diff
@@ -238,6 +276,51 @@ $ opam install dune merlin ocaml-lsp-server odoc ocamlformat utop
 
 Now that the tools are installed, it remains to understand how to use them. Most of them will be driven either by the editor or by Dune, but UTop is handy to try OCaml or a specific library.
 
+## Up and Running with the Platform Installer
+
+As of 2023, the [OCaml Platform
+Installer](https://github.com/tarides/ocaml-platform-installer), presented here,
+is new recommended way to install OCaml in Unix. As of any freshly released
```

Comment:
I'm afraid this isn't quite how the 'incubated' projects are intended to be used in the Platform categorisation.  If the OCaml Platform installer isn't stable (as is indicated above in the 'installing on Unix' section), then it should not be the recommended way for users to install something on their system.  We should be recommending what actually works, even if it's cumbersome.

The 'incubated' projects should be marked as such, along with channels for feedback so that if users opt-in to try it, they can give feedback.  For a busy user though, the existing mechanisms (e.g. `brew install opam`) should be what we recommend since they are known to work reliably.

Reply:
Thanks, I will update `platform.eml` accordingly too.